### ISSUE TITLE
General guidance for translators

### ISSUE THREAD

**Issue by christianpeel** [PiensaEnJulia/PiensaEnJulia.jl#5]
Here are a few suggestions things to keep in mind while translating. We'll update this issue as time goes on.

* It's fine to use Google translate to help with a translation. While this can be a big help with the text, Google translate will also translate AsciiDoc keywords that are derived from English into Spanish, which is bad. For example, the word source in "`[source,...]`" is a keyword, and should not be translated.
* The keywords  "[TIP]", "[WARNING]", "[IMPORTANT]", and "[CAUTION]" should be kept in English.
* The [decimal separator](https://en.wikipedia.org/wiki/Decimal_separator)  used in Julia is the period. So even though in various Spanish-speaking locales you would write 84/2 as "42,0", Julia uses "42.0". Yes, a localization package could fix this, but that's a project outside the scope of PiensaEnJulia.
* If in doubt about the correct Spanish computer science word to use, one can look to the two translations of Think Python ([one](www.openbookproject.net/thinkcs/archive/python/spanish2e/cap02.html), [two](https://argentinaenpython.com/quiero-aprender-python/aprenda-a-pensar-como-un-programador-con-python.pdf)). 
* ThinkJulia is a port of "ThinkPython" which uses many Monty Python references. We've [discussed](https://github.com/PiensaEnJulia/PiensaEnJulia.jl/issues/3) removing these.
* When translating an English Idiom, look for something that will be as widely understood in the Spanish-speaking world as possible. 
* As well as the .asciidoc files, the Julia files in the [src/code](https://github.com/PiensaEnJulia/PiensaEnJulia.jl/tree/master/src/code) and [src/solutions](https://github.com/PiensaEnJulia/PiensaEnJulia.jl/tree/master/src/solutions) directories need to be translated.

**Comment by alejandromerchan** [comment]
My first impression regarding the first bullet point is that there's a mix between the use of "tu" and "usted", with some paragraphs using one or the other. I think we should consistently use "usted" and its conjugations.

**Comment by pambus** [comment]
Ok, I was unsure about the use of one or the other. Thank you for the suggestion!.

**Comment by lventosa** [comment]
Yeah, I think that the "usted" form is more inclusive considering all the variants of the Spanish language around the world.
Also, regarding the "I" or "we" issue we should try to stick to more impersonal forms such as "se cambiaron" instead of "cambié"/"cambiamos". However, this brings another issue regarding verbal tenses. In some parts of the book there's a strong use of "pretérito perfecto compuesto o antepresente" (he cambiado) instead of the "pretérito perfecto simple" (cambié). I think we should stick to the latter as, again, it's much more inclusive regarding all the variants of the Spanish language.

**Comment by christianpeel** [comment]
Por cierto, puedes hacer nuevos "issues" en español. Puedo usar Google Translate si es necesario :-)

**Comment by smendozabarrera** [comment]
Hi guys, peased to meet you, I see much more activity after the JuliaCon, that is great, I made a few suggestions, what is the better way to send it? Le me know, regards!

**Comment by lventosa** [comment]
Hey! You can just open a pull request (after forking the repo and adding your changes) and we will review your suggestions. If approved the changes will be merged into the previous work. Thank you!!
